base_transaction_id/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2012 Yannick Vaucher, Camptocamp SA
```

Comment:
2012-2018 ? Same for other headings

Reply:
https://github.com/OCA/account-reconcile/pull/190/commits/18615b14d20c81bd58a02681713aee58adb7caff

Here are the changes I made, I don't think it deserve a bump on copyright as no code was change in python files.